drivers/wifi/nrfwifi/src/net_if.c

```diff
@@ -158,6 +159,7 @@ static void nrf_wifi_rpu_recovery_work_handler(struct k_work *work)
 	}
 	rpu_ctx_zep->rpu_recovery_in_progress = false;
 	rpu_ctx_zep->last_rpu_recovery_time_ms = k_uptime_get();
+	rpu_ctx_zep->rpu_recovery_success++;
```

Comment:
Even for failure we are continuing above #148, so, need to adjust this counter only for successful case.